pkg/client/keycloak/config.go

```diff
@@ -9,12 +9,8 @@ import (
 	"github.com/spf13/pflag"
 )
 
-type SSOProvider string
-
 const (
-	MAS_SSO                       SSOProvider = "mas_sso"
-	REDHAT_SSO                    SSOProvider = "redhat_sso"
```

Comment:
Why this change?

Reply:
Same as https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1007#discussion_r866635039